### PR TITLE
Add react navigation 1

### PR DESCRIPTION
**Pull request by KhaninPasha0488**

### CODE REVIEW

**Review by HuziMuzi — APPROVED** [review]

File: App.tsx [hunk 1]
```diff
@@ -17,16 +26,35 @@ const App = () => {
   }, []);
 
   return (
-    <Provider store={store}>
```
Comment (antoniosk10) [inline App.tsx:20]: Why did you delete this?

File: android/app/src/main/java/com/ccmobile/MainActivity.java [hunk 2]
```diff
@@ -4,13 +4,17 @@
...
+import android.os.Bundle;
 
 public class MainActivity extends ReactActivity {
-@Override
+   /**
+    * React Navigation
+    */
+    @Override
     protected void onCreate(Bundle savedInstanceState) {
-        SplashScreen.show(this);  // here
```
Comment (antoniosk10) [inline android/app/src/main/java/com/ccmobile/MainActivity.java:11]: Why did you delete this?

Comment (KhaninPasha0488) [reply android/app/src/main/java/com/ccmobile/MainActivity.java:11]: did as in the docs
Снимок экрана 2022-12-16 в 14 48 05

Comment (antoniosk10) [reply android/app/src/main/java/com/ccmobile/MainActivity.java:11]: But you broke splash screen functionality. You need to return this code. And check does splash screen work again.

**Review by Julia-Kovalchuk — APPROVED** [review]

**Review by antoniosk10 — APPROVED** [review]
